docs/reference/single-demonstration-grasping.md

```diff
@@ -161,4 +161,55 @@ simply run:
 5. $ ./single_demo_inference.py
 ```
 
+## Performance Evaluation
 
+TABLE-1: OpenDR Single Demonstration Grasping platform inference speeds.
+| Platform              | Inference speed (FPS)  |
+| --------------------- | ---------------------- | 
+| Nvidia GTX 1080 ti    | 20                     |
+| Nvidia Geforce 940mx  | 2.5                    | 
+| Jetson Xavier NX      | 4                      | 
+| CPU                   | 0.4                    | 
+
+
+
+The energy consumption of the detection model during inference was also measured on Xavier NX and reported accordingly. It is worth mentioning that the inference on the first inference iteration requires more energy for initialization which as it can be seen in TABLE-2.
```

Comment:
```suggestion
The energy consumption of the detection model during inference was also measured on Xavier NX and reported accordingly.
It is worth mentioning that the inference on the first inference iteration requires more energy for initialization which as it can be seen in TABLE-2.
```